diagnostic.md

```diff
@@ -53,15 +75,17 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+1. .big{}
+2. #contentPane {}
+3. .important.red{}
+4. p.div {}
```

Comment:
`div > p { }` will be a child relationship. When you don't have a space, it is saying assign the below styles to elements with BOTH the classes listed. when there is a space, it is a decendent and when its a '>' it's a child.